dev/com.ibm.ws.concurrent.persistent_fat_simctrl/publish/files/features/controllerTestFeature-1.0.mf

```diff
@@ -0,0 +1,9 @@
+Subsystem-ManifestVersion: 1
+IBM-ShortName: controllerTestFeature-1.0
```

Comment:
This feature file should have the `IBM-Test-Feature: true` line in it somewhere to indicate that it's a test feature which acts as a safety net to make sure we never accidentally ship it